scripts/setup.py

```diff
@@ -82,9 +82,23 @@ def download_and_install_package(url_to_retrieve, filename):
     tar = tarfile.open(name=filename)
     tar.extractall(path=common.OPPIA_TOOLS_DIR)
     tar.close()
+    rename_folder(filename, common.OPPIA_TOOLS_DIR)
     os.remove(filename)
 
 
+def rename_folder(filename, path):
```

Comment:
The function name should be more specific. `rename_yarn_folder` maybe?

Reply:
I actually thought that this function could be used in the future if some other module had a similar problem, ergo the general name, but I'll rename it to `rename_yarn_folder` for now.